rabies/analysis_pkg/cpca/modeling.py

```
@@ -0,0 +1,203 @@
+
+
+def cpca_auto(X, C_prior, N_max, Wt_n='n', min_prior_sim=None, Dc_W_thresh=None, Dc_C_thresh=None):
+    '''
+    Conduct CPCA with automated estimation of n*. Wt_n sets a maximum for how many components are used for temporal CPCA.
+    '''
+    
+    if Wt_n=='n':
+        Wt_n = N_max
+    elif not type(Wt_n) is int:
+        raise ValueError("Wt_n must be an integer or 'n'.") 
+    elif Wt_n<0:
```

Flake8 (7.2.0)

[error] missing whitespace around operator

(E225)